portfolio/.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Delete this from this branch.  The linter stuff should be submitted independently